### PR TITLE
fix ex 5

### PR DESCRIPTION
**Pull request by at-tanhle**

### CODE REVIEW

File: CustomView/Fix_Exercise_5/Exercise_5/HomeViewController.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        super.viewDidLoad()
+        
+        myScrollView = UIScrollView(frame: UIScreen.main.bounds)
+        myScrollView.isScrollEnabled = true
+        let distance = (myScrollView.bounds.width - 300) / 4
+        let viewWidth: CGFloat = 100
+        let viewHeith: CGFloat = 120
+        var originX: CGFloat = distance
+        var originY: CGFloat = 30
+        var scrollViewContentSize: CGFloat = 150
```
Comment (at-truongnguyen) [inline CustomView/Fix_Exercise_5/Exercise_5/HomeViewController.swift]: `contentSize` but type `CGFloat`? I think `contentSize` is a `CGSize` one. or you should change to `contentSizeHeight`

File: CustomView/Fix_Exercise_5/Exercise_5/HomeViewController.swift [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        userView.avatarImageView.image = #imageLiteral(resourceName: "no_avatar")
+    }
+}
+extension HomeViewController: UserViewDelegate {
+    func didSelectUserView(userView: UserView) {
+        if let name = userView.nameLabel.text {
+            print(name)
+        } else {
+            return
+        }
```
Comment (at-truongnguyen) [inline CustomView/Fix_Exercise_5/Exercise_5/HomeViewController.swift]: Do we really need to return?

File: CustomView/Fix_Exercise_5/Exercise_5/HomeViewController.swift (hunk 1 above)
Comment (at-tanhle) [inline CustomView/Fix_Exercise_5/Exercise_5/HomeViewController.swift]: OK.I'm gonna fix it.

File: CustomView/Fix_Exercise_5/Exercise_5/HomeViewController.swift (hunk 2 above)
Comment (at-tanhle) [inline CustomView/Fix_Exercise_5/Exercise_5/HomeViewController.swift]: I got it

**Pull request merged**